src/components/CurrencyLogoBridge/index.tsx

```diff
@@ -0,0 +1,135 @@
+import { ChainId, Currency, Token, DXD, SWPR } from '@swapr/sdk'
+import React, { ReactNode, useMemo } from 'react'
+import Skeleton from 'react-loading-skeleton'
+import styled from 'styled-components'
+
+import EtherLogo from '../../assets/images/ether-logo.png'
+import XDAILogo from '../../assets/images/xdai-logo.png'
+import DXDLogo from '../../assets/svg/dxd.svg'
+import SWPRLogo from '../../assets/images/swpr-logo.png'
+import carrotListLogoUrl from '../../assets/images/carrot.png'
+import { useActiveWeb3React } from '../../hooks'
+import useHttpLocations from '../../hooks/useHttpLocations'
+import { WrappedTokenInfo } from '../../state/lists/wrapped-token-info'
+import Logo from '../Logo'
+import { useBridgeTokenInfo } from '../../services/EcoBridge/EcoBridge.hooks'
+
+const getTokenLogoURL = (address: string) =>
+  `https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/${address}/logo.png`
+
+const StyledLogo = styled(Logo)<{ size: string }>`
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  width: ${({ size }) => size};
+  height: ${({ size }) => size};
+  border-radius: ${({ size }) => size};
+`
+// looading is for purpose, for some reason DevTools screams that 'loading' is expected to be a string
+const Wrapper = styled.div<{ size: string; marginRight: number; marginLeft: number; looading?: boolean }>`
+  position: relative;
+  width: ${({ size }) => size};
+  height: ${({ size }) => size};
+  margin-right: ${({ marginRight }) => marginRight}px;
+  margin-left: ${({ marginLeft }) => marginLeft}px;
+  border-radius: ${({ size }) => size};
```

Comment:
Destruct these from the top 

```suggestion
const Wrapper = styled.div<{ size: string; marginRight: number; marginLeft: number; looading?: boolean }>(({ size, marginRight, marginLeft }) =>`
  position: relative;
  width: ${size};
  height: ${size};
  margin-right: ${marginRight}px;
  margin-left: ${marginLeft}px;
  border-radius: ${size};
```